libweasyl/libweasyl/models/tables.py

```diff
@@ -355,8 +355,8 @@ def default_fkey(*args, **kwargs):
     Column('login_name', String(length=40), nullable=False, unique=True),
     Column('hashpass', String(length=100), nullable=False),
     Column('email', String(length=100), nullable=False, unique=True),
-    Column('birthday', WeasylTimestampColumn(), nullable=False),
-    Column('unixtime', WeasylTimestampColumn(), nullable=False),
+    Column('birthday', TIMESTAMP(timezone=True), nullable=False, server_default=func.now()),
```

Comment:
I think `birthday` should be a `date`, so we might want to save it for a separate PR.

Reply:
That almost makes too much sense...
Reverted, will play with it, and make a pull request for that later :)